### PR TITLE
labels for data-broker

### PR DESCRIPTION
**Pull request by rssag**

### CODE REVIEW

**Review by MWindel — APPROVED** [review]

File: content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md [hunk 1]
```diff
@@ -26,21 +26,21 @@ In summary, to work with the microservice-based data broker, the following requi
 {{< /c8y-admon-req >}}
 
 
-### To enable the microservice-based data broker
+### To enable the microservice-based data broker {#to-enable-the-microservicebased-data-broker}
```
Comment (BeateRixen) [inline content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md:29]: > microservice-based

File: content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md [hunk 2]
```diff
@@ -26,21 +26,21 @@ In summary, to work with the microservice-based data broker, the following requi
...
 
 
-###  Data subscriptions
+### Data subscriptions {#data-subscriptions}
 
 See [Data subscriptions](/data-broker/data-broker-application/#data-subscriptions) for details on how to manage data subscriptions.
 
 
-###  Migrating existing data connectors to the microservice-based data broker
+###  Migrating existing data connectors to the microservice-based data broker {#-migrating-existing-data-connectors-to-the-microservicebased-data-broker}
```
Comment (BeateRixen) [inline content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md:44]: > microservice-based

File: content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md (hunk 1 above)
Comment (rssag) [inline content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md:29]:
```suggestion
### To enable the microservice-based data broker {#to-enable-the-microservice-based-data-broker}
```

File: content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md (hunk 2 above)
Comment (rssag) [inline content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md:44]:
```suggestion
###  Migrating existing data connectors to the microservice-based data broker {#-migrating-existing-data-connectors-to-the-microservice-based-data-broker}
```

File: content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md (hunk 1 above)
Comment (rssag) [reply content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md]: added

File: content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md (hunk 2 above)
Comment (rssag) [reply content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md]: added

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**